### PR TITLE
Updated post 1903 log location

### PR DESCRIPTION
**Pull request by jeffgilb**
These logs moved in 1903.

### CODE REVIEW

**Comment by e0i** [comment]
@greg-lindsay The PR that expands the description on Windows Autopilot log location is available for review. Thanks.

File: windows/deployment/windows-autopilot/troubleshooting.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ If the expected Autopilot behavior does not occur during the out-of-box experien
 
 ### Windows 10 version 1803 and above
 
-To see details related to the Autopilot profile settings and OOBE flow, Windows 10 version 1803 and above adds event log entries.  These can be viewed using Event Viewer, navigating to the log at **Application and Services Logs –> Microsoft –> Windows –> Provisioning-Diagnostics-Provider –> AutoPilot**.  The following events may be recorded, depending on the scenario and profile configuration.
+To see details related to the Autopilot profile settings and OOBE flow, Windows 10 version 1803 and above adds event log entries.  These can be viewed for Windows 10 1903 or later using Event Viewer, and navigating to the log at **Applications and Services Logs -> Microsoft -> Windows -> ModernDeployment-Diagnostics-Provider -> Autopilot**. In previous versions of Windows 10, the event log is located at **Applications and Services Logs –> Microsoft –> Windows –> Provisioning-Diagnostics-Provider –> AutoPilot**. The following events may be recorded, depending on the scenario and profile configuration.
```
Comment (illfated) [inline windows/deployment/windows-autopilot/troubleshooting.md]: Improved focus on Windows 10 version + more natural language.
```suggestion
To see details related to the Autopilot profile settings and OOBE flow, Windows 10 version 1803 and above adds event log entries.  For Windows 10 1903 or later, these can be viewed by using Event Viewer and navigating to the log at **Applications and Services Logs -> Microsoft -> Windows -> ModernDeployment-Diagnostics-Provider -> Autopilot**. In previous versions of Windows 10, the event log is located at **Applications and Services Logs –> Microsoft –> Windows –> Provisioning-Diagnostics-Provider –> AutoPilot**. The following events may be recorded, depending on the scenario and profile configuration.
```

**Comment by e0i** [comment]
@greg-lindsay Following up for updates.

**Comment by illfated** [comment]
And if Greg is busy with lots of other work assignments, maybe @lizap could help out with final review and merge the PR if all is well?

**Comment by illfated** [comment]
The only other change I can think of (although far from required), is to add separate bullet points both for the 1903 and the 1803 location description within this PR content. Then again, looking at the rest of the affected page, it could result in an overuse of bullet points, making it look like almost all the descriptive text uses bullet points.

**Comment by e0i** [comment]
@greg-lindsay @martyav Would you be able to review this PR as the time permits? Thanks.

**Comment by e0i** [comment]
@greg-lindsay @martyav Following up for updates.

**Comment by illfated** [comment]
@jeffgilb : Are you able to resolve the conflict between your PR file content and the master branch file `troubleshooting.md` ? Or would you like some help with that?

**Comment by e0i** [comment]
@jeffgilb Would you be able to resolve merge conflicts with the file `troubleshooting.md` ? We would like to merge your contributions. Thanks.

**Comment by jeffgilb** [comment]
It looks like the entire article is a conflict now somehow?!

**Comment by illfated** [comment]
Yes, this is because someone else updated the same page while you were waiting for your PR to be reviewed. These things happen some times, so we need to deal with this type of change too.

**Comment by e0i** [comment]
@jeffgilb Please let us know whenever the conflicts are resolved. Thanks.

**Comment by illfated** [comment]
Feel free to ask for help, in case you find the conflict difficult or too hard to resolve.

**Comment by jeffgilb** [comment]
I’m in training all week until around 8pm everyday. If there’s a way to easily resolve this, please do. I only changed one small section so not sure why the entire thing is conflicting now and don’t really have the bandwidth to figure out completely what is going on.

Thanks
~Jeff

**Comment by illfated** [comment]
The reason PR files get conflicts is because there will always be someone else will find it necessary to change some other part of the file, and that change can be merged into the source code before your suggested change makes it there. That happens a lot, because of the hundreds, maybe thousands of people working on the very same source tree as you do. Some times, housekeeping of a long list of files is done by admins, or the PR content file you are working on may simply be on hold for another index change while you are still waiting for your change to become accepted by the admins. Meanwhile, conflicts occur, because you have not kept your personal fork of the repository up to date with the Microsoft source (your fork, [`jeffgilb/windows-itpro-docs`](https://github.com/jeffgilb/windows-itpro-docs) is currently 447 commits behind [`MicrosoftDocs:master`](https://github.com/MicrosoftDocs/windows-itpro-docs)).

**Comment by illfated** [comment]
The simple explanation is that 1 line got inserted between line 11 and 12 (addition of a New line 12) in the target file (https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/deployment/windows-autopilot/troubleshooting.md):
```diff
 ---
 title: Troubleshooting Windows Autopilot
 description: Windows Autopilot deployment
 keywords: mdm, setup, windows, windows 10, oobe, manage, deploy, autopilot, ztd, zero-touch, partner, msfb, intune
 ms.reviewer: mniehaus
 manager: laurawi
 ms.prod: w10
 ms.mktglfcycl: deploy
 ms.localizationpriority: medium
 ms.sitesec: library
 ms.pagetype: deploy
+audience: itpro
 author: greg-lindsay
 ms.author: greglin
 ms.collection: M365-modern-desktop
 ms.topic: article
 ---
```

**Comment by illfated** [comment]
Also worth noticing: Besides the PR author (you), it will only be possible for the Microsoft Docs team administrators (authors and managers) to merge, edit and make changes to the PR content. All the rest of us can only comment here in the dialogue page or in the code, suggest changes via code comments or open pull requests ourselves. (Although, if you think it will be any easier to accept a pull request into your own repository than to edit your file to add a new line number 12 with my highlighted text above, feel free to tell me and I will clone your repository to my computer to create the PR for this change.

**Comment by jeffgilb** [comment]
Ok, if you can’t make the merge happen, just close the PR.
If you need to do that then I’ll just make a new one when I get time in the private repo.

**Comment by e0i** [comment]
> Ok, if you can’t make the merge happen, just close the PR. If you need to do that then I’ll just make a new one when I get time in the private repo.

Closing as suggested. Thank you for taking the time to contribute to the repository.

**Comment by e0i** [comment]
@officedocsbot close

**Pull request closed without merging**